tests/guest-tools/win/test_guest_tools_win.py

```diff
@@ -0,0 +1,346 @@
+import enum
+import logging
+import pytest
+import time
+from lib.common import wait_for
+from lib.snapshot import Snapshot
+from lib.vm import VM
+from . import *
+
+
```

Comment:
We describe the requirements on top of test files : pools, hosts, VMs, additional resources, specific configurations...